### PR TITLE
ref(escalating issues): Remove front end flag usage

### PR DESCRIPTION
**Pull request by ceorourke**
Remove front end usages of the `escalating-issues` flag - this has been GA'd since [August](https://github.com/getsentry/getsentry/pull/11542)/[September](https://github.com/getsentry/getsentry/pull/11627).

### CODE REVIEW

File: static/app/types/hooks.tsx [hunk 1]
```diff
@@ -170,7 +170,6 @@ export type ComponentHooks = {
   'component:disabled-member': () => React.ComponentType<DisabledMemberViewProps>;
   'component:disabled-member-tooltip': () => React.ComponentType<DisabledMemberTooltipProps>;
   'component:enhanced-org-stats': () => React.ComponentType<OrganizationStatsProps>;
-  'component:escalating-issues-banner-feedback': () => React.ComponentType<QualitativeIssueFeedbackProps>;
```
Comment (ceorourke) [inline static/app/types/hooks.tsx:173]: Do we still want this? I'm looking at the getsentry usages of the flag too and am not sure if I should remove the feedback banner.

**Comment by ceorourke** [comment]
I don't do much front end so I may have done some things wrong here! Please lmk and I will fix

File: static/app/components/eventOrGroupHeader.tsx [hunk 2]
```diff
@@ -60,8 +60,7 @@ function EventOrGroupHeader({
       <Fragment>
         {!hideLevel && level && <GroupLevel level={level} />}
         {!hideIcons &&
-          status === 'ignored' &&
-          !organization.features.includes('escalating-issues') && (
+          status === 'ignored' && (
```
Comment (snigdhas) [inline static/app/components/eventOrGroupHeader.tsx:63]: if my boolean logic is right, we should remove this block since `!organization.features.includes('escalating-issues')` would've been returning `False`?

File: static/app/components/eventOrGroupHeader.tsx [hunk 3]
```diff
@@ -89,12 +88,11 @@ function EventOrGroupHeader({
   function getTitle() {
     const {id, status} = data as Group;
     const {eventID: latestEventId, groupID} = data as Event;
-    const hasEscalatingIssues = organization.features.includes('escalating-issues');
 
     const commonEleProps = {
       'data-test-id': status === 'resolved' ? 'resolved-issue' : null,
       style:
-        status === 'resolved' && !hasEscalatingIssues
+        status === 'resolved'
```
Comment (snigdhas) [inline static/app/components/eventOrGroupHeader.tsx:95]: Same thing here, `!hasEscalatingIssues` is False

File: static/app/views/issueDetails/actions/index.tsx [hunk 4]
```diff
@@ -380,7 +376,7 @@ export function Actions(props: Props) {
           size: 'sm',
         }}
         items={[
-          ...(isIgnored || hasEscalatingIssues
+          ...(isIgnored
```
Comment (snigdhas) [inline static/app/views/issueDetails/actions/index.tsx:379]: Can we remove this whole block? `isIgnored || hasEscalatingIssues` -> `isIgnored || True` -> `True`?

File: static/app/views/issueDetails/groupActivityItem.tsx [hunk 5]
```diff
@@ -98,10 +98,9 @@ function GroupActivityItem({
   author,
 }: GroupActivityItemProps) {
   const issuesLink = `/organizations/${organization.slug}/issues/`;
-  const hasEscalatingIssuesUi = organization.features.includes('escalating-issues');
 
   function getIgnoredMessage(data: GroupActivitySetIgnored['data']) {
-    const ignoredOrArchived = hasEscalatingIssuesUi ? t('archived') : t('ignored');
+    const ignoredOrArchived = t('archived');
```
Comment (snigdhas) [inline static/app/views/issueDetails/groupActivityItem.tsx:103]: let's rename for clarity

```suggestion
    const archived = t('archived');
```

File: static/app/views/issueDetails/header.tsx [hunk 6]
```diff
@@ -266,10 +265,10 @@ function GroupHeader({
               <h3>
                 <StyledEventOrGroupTitle data={group} />
               </h3>
-              {!hasEscalatingIssuesUi && group.inbox && (
+              {group.inbox && (
                 <InboxReason inbox={group.inbox} fontSize="md" />
               )}
```
Comment (snigdhas) [inline static/app/views/issueDetails/header.tsx:270]: This block should get deleted

File: static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx [hunk 7]
```diff
@@ -16,7 +16,7 @@ describe('TeamIssuesBreakdown', () => {
       url: `/teams/${organization.slug}/${team.slug}/issue-breakdown/`,
       body: TeamIssuesBreakdownFixture(),
     });
-    const statuses = ['new', 'regressed', 'unignored'];
+    const statuses = ['new', 'regressed', 'unarchived'];
```
Comment (snigdhas) [inline static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx:19]: do we support unarchived as a status? 🤔

File: static/app/views/settings/organizationDeveloperSettings/subscriptionBox.tsx [hunk 8]
```diff
@@ -54,9 +54,7 @@ function SubscriptionBox({
 
   const DESCRIPTIONS: Record<(typeof EVENT_CHOICES)[number], string> = {
     // Swap ignored for archived if the feature is enabled
-    issue: `created, resolved, assigned, ${
-      features.includes('escalating-issues') ? 'archived' : 'ignored'
-    }`,
+    issue: `created, resolved, assigned, ${'archived'}`,
```
Comment (snigdhas) [inline static/app/views/settings/organizationDeveloperSettings/subscriptionBox.tsx:57]:
```suggestion
    issue: `created, resolved, assigned, archived`,
```

File: static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx (hunk 7 above)
Comment (ceorourke) [reply static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx:19]: idk 😓 what's the new version of unignored? escalating?

Comment (snigdhas) [reply static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx:19]: ah yeah, escalating is probably more accurate!

File: static/app/types/hooks.tsx (hunk 1 above)
Comment (ceorourke) [reply static/app/types/hooks.tsx:173]: For posterity, I asked and Brian said to keep it for now.

File: static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx (hunk 7 above)
Comment (ceorourke) [reply static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx]: hmm now I'm confused - I'm getting this typing error:

```
Error: static/app/views/organizationStats/teamInsights/issues.tsx(111,48): error TS2322: Type '"escalating"' is not assignable to type '"new" | "deleted" | "ignored" | "resolved" | "regressed" | "archived" | "unarchived" | "unignored"'.
```

and it looks like `unarchived` is defined [here](https://github.com/getsentry/sentry/blob/master/static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx#L21-L31) but `escalating` is not - I don't have any context around this, do you have any idea what's correct?

Comment (snigdhas) [reply static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx]: hm idk since this is some stats breakdown - "unarchive" is probably correct there then!

File: static/app/components/eventOrGroupExtraDetails.tsx [hunk 9]
```diff
@@ -54,14 +54,13 @@ function EventOrGroupExtraDetails({
 
   const showReplayCount =
     organization.features.includes('session-replay') && projectCanLinkToReplay(project);
-  const hasEscalatingIssuesUi = organization.features.includes('escalating-issues');
 
   return (
     <GroupExtra>
-      {!hasEscalatingIssuesUi && inbox && (
+      {inbox && (
```
Comment (snigdhas) [inline static/app/components/eventOrGroupExtraDetails.tsx:60]: oops missed this earlier, but the `!hasEscalatingIssuesUi` block can be deleted.

File: static/app/components/eventOrGroupHeader.tsx [hunk 10]
```diff
@@ -89,14 +82,10 @@ function EventOrGroupHeader({
...
     const {eventID: latestEventId, groupID} = data as Event;
-    const hasEscalatingIssues = organization.features.includes('escalating-issues');
 
     const commonEleProps = {
       'data-test-id': status === 'resolved' ? 'resolved-issue' : null,
-      style:
-        status === 'resolved' && !hasEscalatingIssues
-          ? {textDecoration: 'line-through'}
-          : undefined,
+      style: undefined,
```
Comment (snigdhas) [inline static/app/components/eventOrGroupHeader.tsx:88]: I also don't do much FE (lol), can we just remove the `style: undefined`? cc @scttcper

File: static/app/views/issueList/utils.tsx [hunk 11]
```diff
@@ -105,7 +100,7 @@ export function getTabs(organization: Organization) {
         name: t('Ignored'),
         analyticsName: 'ignored',
         count: true,
-        enabled: !hasEscalatingIssuesUi,
+        enabled: !true,
```
Comment (snigdhas) [inline static/app/views/issueList/utils.tsx:103]:
```suggestion
        enabled: false,
```

Comment (ceorourke) [reply static/app/views/issueList/utils.tsx:103]: lol

**Review by snigdhas — APPROVED** [review]

File: static/app/components/feedback/feedbackItem/crashReportSection.tsx [hunk 12]
```diff
@@ -46,7 +45,6 @@ export default function CrashReportSection({
         data={groupData}
         size="normal"
       />
-      <EventOrGroupExtraDetails data={groupData} showInboxTime />
```
Comment (ryan953) [inline static/app/components/feedback/feedbackItem/crashReportSection.tsx:49]: why is this removed?

**Review by billyvg — APPROVED** [review]

File: static/app/components/feedback/feedbackItem/crashReportSection.tsx (hunk 12 above)
Comment (ceorourke) [reply static/app/components/feedback/feedbackItem/crashReportSection.tsx]: put it back 👍
